core/controllers/editor.py

```diff
@@ -563,13 +563,11 @@ def get(self, exploration_id, escaped_state_name):
 
                 trained_answers = set()
                 for answer_group in interaction.answer_groups:
-                    for rule_spec in answer_group.rule_specs:
-                        if (rule_spec.rule_type ==
-                                exp_domain.RULE_TYPE_CLASSIFIER):
```

Comment:
I'd expect this constant to be deleted altogether. Did we do that? (I don't see it.)

Reply:
Ah no we need it because of migration from v23 to v24, it is required to check the classifier rule and get training data.

Reply:
Ah ok. Is that the only place that it's used?

If so, I would suggest removing it and hardcoding the string in the conversion method, since we explicitly don't want it to be reused.

if not, we should add a comment above the constant to say that it's deprecated and kept only for historical purposes.

Reply:
Yes that is the only place it is being used right now.

Reply:
Done